### PR TITLE
[11.0][MIG] stock_picking_back2draft

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Nothing to do

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
v12 or v11? commit says 12 but is pointing to v11 branch.

**Comment by pedrobaeza** [comment]
Why do you need this? Unlock the picking and modify whatever...

**Comment by pedrobaeza** [comment]
In fact, this can lead to some inconsistencies in current flows.

**Comment by rousseldenis** [comment]
@pedrobaeza In some situations, people cancelled pickings (manipulation error or ....). If you cannot set it back to draft, it is impossible easily to solve the situation

**Comment by pedrobaeza** [comment]
OK, if you think there's no problem on this, go ahead.

**Comment by AaronHForgeFlow** [comment]
> v12 or v11? commit says 12 but is pointing to v11 branch.

v11? :smile: 

> Why do you need this? Unlock the picking and modify whatever...

Hi @pedrobaeza  I did not notice about that unlock option, is it standard?

> In fact, this can lead to some inconsistencies in current flows.

Same as in previous versions of the module, but it helps to fix cancellation by mistake...

**Comment by pedrobaeza** [comment]
Yes, the unlock option is standard. I don't remember if it's enabled by default or you need to enable it on the settings. But this option is on done pickings, not on cancelled ones.

**Comment by rousseldenis** [comment]
@aheficent What's the status of this ?

**Review by robyf70 — APPROVED** [review]

**Comment by AaronHForgeFlow** [comment]
@rousseldenis I think this is still ready to review. The objective of this module is to allow quick fixes by users when cancelling stock pickings, same as in previous versions. I think it is still valid.

**Review by rousseldenis — APPROVED** [review]
Code review

File: stock_picking_back2draft/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# © 2016 Lorenzo Battistini - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Pickings back to draft",
+    "summary": "Reopen cancelled pickings",
+    "version": "11.0.1.0.0",
+    "category": "Warehouse Management",
+    "website": "https://www.agilebg.com",
```
Comment (rousseldenis) [inline stock_picking_back2draft/__manifest__.py]: @aheficent Could you change website to OCA repo ?

Comment (AaronHForgeFlow) [reply stock_picking_back2draft/__manifest__.py]: Sure, done now. Thanks for reviewing.

**Review by HaraldPanten — APPROVED** [review]
Tested on runbot. Functional review 👍

**Pull request merged**